.github/workflows/rlog.yaml

```diff
@@ -0,0 +1,24 @@
+name: rlog
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: build
+      run: ./build.sh
+    - name: zip bin directory
+      run: | 
+        cd build
+        sudo tar -czf bin.tar.gz bin
+    - name: Upload bin.tar.gz to release
```

Comment:
gate this on push and on the branch name "rlog-release", this runs twice currently because of the push and pull request triggers